contrib/pg_tde/documentation/docs/variables.md

```diff
@@ -4,44 +4,55 @@ The `pg_tde` extension provides GUC variables to configure the behaviour of the
 
 ## pg_tde.wal_encrypt
 
+**Type** - boolean <br>
+**Default** - off
+
 A `boolean` variable controlling if WAL writes are encrypted or not.
 
 Changing this variable requires a server restart, and can only be set at the server level.
 
-WAL encryption is controlled globally, if enabled all WAL writes are encrypted.
-
-This variable only controls new writes to the WAL, it doens't affect existing WAL records.
-`pg_tde` is always capable of reading existing encrypted WAL records, as long as the keys used for the encryption are still available.
+WAL encryption is controlled globally. If enabled, all WAL writes are encrypted in the entire PostgreSQL cluster.
 
+This variable only controls new writes to the WAL, it doesn't affect existing WAL records.
 
+`pg_tde` is always capable of reading existing encrypted WAL records, as long as the keys used for the encryption are still available.
 
-Enabling WAL encryption requires a configured global principal key.
-
-The default value is false.
+Enabling WAL encryption requires a configured global principal key. Refer to the [WAL encryption configuration](wal-encription.md) documentation for more information.
 
 ## pg_tde.enforce_encryption
 
-A `boolean` variable controlling if creation of new not encrypted tables is enabled or not.
+**Type** - boolean <br>
+**Default** - off
 
-This can be set at global, database, user or session level, but changing the value requires superuser permissions.
```

Comment:
We have the same sentence for the next variable too, and now they are different. These should be consistent.

Reply:
Sure, I'll update it once I'm done with the one above

Reply:
Updated